Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBoxItem.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A control that manages as the item logic for the <see cref="TokenizingTextBox"/> control.
+    /// </summary>
+    [TemplatePart(Name = PART_ClearButton, Type = typeof(Button))]
+    public class TokenizingTextBoxItem : Button
+    {
+        private const string PART_ClearButton = "PART_ClearButton";
+
+        private Button _clearButton;
+
+        /// <summary>
+        /// Event raised when the 'Clear' Button is clicked.
+        /// </summary>
+        public event TypedEventHandler<TokenizingTextBoxItem, RoutedEventArgs> ClearClicked;
```

Comment:
Huh, would have never thought of 'Clear' for a name, but that is what the symbol is called...

Wondering if we need this bubbled to the parent control as well like we did with the TabViewItem's so the TabView control owner can more easily intercept & prevent the user from removing items (i.e. if they want an are you sure or to have certain things not removable).

Probably something we can worry about later though.  NVM.